contracts/golos.worker/main.cpp

```diff
@@ -934,7 +927,7 @@ class worker : public contract
   {
     LOG("proposal_id: %, comment: %", proposal_id, comment.text.c_str());
     auto proposal_ptr = get_proposal(proposal_id);
-    eosio_assert(proposal_ptr->state == proposal_t::STATE_TSPEC_AUTHOR_REVIEW, "invalid proposal state");
+    eosio_assert(proposal_ptr->state == proposal_t::STATE_DELEGATES_REVIEW, "invalid proposal state");
```

Comment:
looks like it's now impossible to change state to `STATE_DELEGATES_REVIEW`, here should be `STATE_WORK`